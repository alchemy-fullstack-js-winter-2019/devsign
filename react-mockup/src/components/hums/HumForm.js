import React from 'react';
import { FormDiv } from './HumStyles';

function HumForm() {
  return (
    <FormDiv>
      <form>
        <label htmlFor="hum"><p>post a new hum</p>
          <input type="text" name="hum" />
        </label>
        <button>add new hum</button>
      </form>
    </FormDiv>
  );
}

export default HumForm;
