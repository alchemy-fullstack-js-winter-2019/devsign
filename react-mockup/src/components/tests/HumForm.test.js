import React from 'react';
import renderer from 'react-test-renderer';
import HumForm from '../HumForm';
import { MemoryRouter } from 'react-router';

describe('HumForm snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <HumForm />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
