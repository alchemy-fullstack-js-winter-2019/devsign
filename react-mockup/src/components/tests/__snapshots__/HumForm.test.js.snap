// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HumForm snapshot can match a snapshot 1`] = `ShallowWrapper {}`;
